### ISSUE TITLE
SSL Support

### ISSUE THREAD

**Issue by johnmcmillion** [jstott/socketio4net#3]
Getting an error initializing handshake when I try to connect using my https node/socket.io connection over SSL.  Is SSL/WSS supported with socketio4net?  Thank you so much!  This library is amazing, we're doing really cool stuff with it, thinking of new applications all the time for our app.  Everything works perfect over plain http.

**Comment by jstott** [comment]
I may have broken something in the move (from codeplex & incorporating the transports), but last check it was all working.  You should be able to trace up through https://github.com/jstott/socketio4net/blob/develop/SocketIO.Client.Net40/Transports/TransportFactory.cs#L16-20 and follow the connection to where the uri is set. 

What sort of error are you getting back (from both client & server perspective)?

**Comment by johnmcmillion** [comment]
Thank you for your message.  It looks good, SSL works with SocketIO4Net too!  I was calling the node server with SocketIO4Net from our QA middleware and had a self-signed cert that I needed to import and set up in the QA middleware cert auth.  The error I was getting was "Error getting handsake from Socket.IO host instance: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel."  Anyhow, it works well now!  Thanks again!
